src/main/java/org/simulator/sbml/SBMLinterpreter.java

```diff
@@ -69,25 +70,7 @@
 import org.simulator.math.odes.EventInProgress;
 import org.simulator.math.odes.FastProcessDESystem;
 import org.simulator.math.odes.RichDESystem;
-import org.simulator.sbml.astnode.ASTNodeInterpreter;
-import org.simulator.sbml.astnode.ASTNodeValue;
-import org.simulator.sbml.astnode.AssignmentRuleValue;
-import org.simulator.sbml.astnode.CompartmentOrParameterValue;
-import org.simulator.sbml.astnode.DivideValue;
-import org.simulator.sbml.astnode.FunctionValue;
-import org.simulator.sbml.astnode.IntegerValue;
-import org.simulator.sbml.astnode.LocalParameterValue;
-import org.simulator.sbml.astnode.MinusValue;
-import org.simulator.sbml.astnode.NamedValue;
-import org.simulator.sbml.astnode.PlusValue;
-import org.simulator.sbml.astnode.PowerValue;
-import org.simulator.sbml.astnode.RateRuleValue;
-import org.simulator.sbml.astnode.ReactionValue;
-import org.simulator.sbml.astnode.RootFunctionValue;
-import org.simulator.sbml.astnode.SpeciesReferenceValue;
-import org.simulator.sbml.astnode.SpeciesValue;
-import org.simulator.sbml.astnode.StoichiometryValue;
-import org.simulator.sbml.astnode.TimesValue;
+import org.simulator.sbml.astnode.*;
```

Comment:
Don't do `*` imports. This is normally not very good style and clutters the namespace.

Reply:
Updated